instruments/abstract_instruments/optical_spectrum_analyzer.py

```diff
@@ -63,7 +63,8 @@ class OpticalSpectrumAnalyzer(Instrument, metaclass=abc.ABCMeta):
 
     # PROPERTIES #
 
-    @abc.abstractproperty
+    @property
+    @abc.abstractmethod
```

Comment:
`@abc.abstractproperty` is deprecated since py3.3. Change to preferred method. Must just be some legacy, since correct in most other places.

Reply:
:+1: